internal/data/daily_questions.go

```diff
@@ -0,0 +1,23 @@
+package data
+
+type DailyQuestions struct {
+	ID            int    `db:"id"`
+	Title         string `db:"title"`
+	TimeForAnswer int    `db:"time_for_answer"`
+	Bounty        int    `db:"bounty"`
+	AnswerOptions string `db:"answer_options"`
+	Active        bool   `db:"active"`
+	StartsAt      int    `db:"starts_at"`
+}
+
+type DailyQuestionsQ interface {
+	New() DailyQuestionsQ
+	Insert(DailyQuestions) error
+	Update(map[string]any) error
+
+	Count() (int64, error)
+	Select() ([]DailyQuestions, error)
+
+	FilteredActive(status bool) DailyQuestionsQ
+	FilteredStartAt(date int) DailyQuestionsQ
```

Comment:
```suggestion
	FilterByActive(status bool) DailyQuestionsQ
	FilterByStartAt(date time.Time) DailyQuestionsQ
```
FilterBy* - select rows which correspond to filter value
Filter* - exclude rows correspond to filter name